src/sentry/api/endpoints/project_searches.py

```diff
@@ -17,6 +17,8 @@ class SavedSearchSerializer(serializers.Serializer):
 
 
 class ProjectSearchesEndpoint(ProjectEndpoint):
+    permission_classes = (ProjectSearchPermission,)
```

Comment:
So we likely dont want to introduce new permissions for this. Searches are a low impact item. We'd need to either:

1. Make the UI respect the existing permissions (e.g. members cant add new saved searches, and definitely cant set team defaults).
2. OR create some unshared saved search behavior, where members can create their own saved searches but they cant share them with the team without having permissions.

I would side with #1, which is generally what the original API implementation was intended to support.